common/seamless.py

```diff
@@ -0,0 +1,150 @@
+import os
+from functools import lru_cache
+import tempfile
+from pydub import AudioSegment
+import math
+import textwrap
+import torchaudio
+
+import requests
+import torch
+from celery.signals import worker_init
+from kombu import Queue
+from seamless_communication.models.inference import Translator
+
+import gooey_gpu
+from api import (
+    SeamlessM4TPipeline,
+    SeamlessM4TInputs,
+    SeamlessM4TOutput,
+)
+from celeryconfig import app, setup_queues
+
+VOCODER_ID = "vocoder_36langs"  # only supported option rn
+
+
+@app.task(name="seamless")
```

Comment:
I'll leave the feedback Dev gave me in case it's helpful:

I think we want a separate method for each type of task (ASR, S2ST, etc.) which would be labeled `@app.task(name="seamless.ASR)`,  `@app.task(name="seamless.T2TT)`, etc.

![image](https://github.com/dara-network/gooey-gpu/assets/97496861/c9baadfc-8a9a-4e84-acbd-c020f34f566e)

I also believe it might be a good idea to remove the tasks we don't use (e.g. "S2ST") to avoid maintaining unused code (which won't be tested)

Reply:
postponed this refactor for later